tsconfig-common.json

```diff
@@ -24,6 +28,6 @@
         // "noImplicitAny": true,
         /* Completeness */
         "skipDefaultLibCheck": true, // it's safe to assume that built-in .d.ts files are correct
-        "skipLibCheck": false
+        "skipLibCheck": true
```

Comment:
**note:** This stops us typechecking node modules.

Reply:
oh I see, that makes sense.